kuma/wiki/jinja2/wiki/includes/spam_error.html

```diff
@@ -14,3 +14,10 @@
     contact the site administrators</a>.
     {% endtrans %}
 </p>
+{% if perms.wiki.add_revisionakismetsubmission %}
```

Comment:
Getting the error "perms is undefined" when this tires to display and the tests are failing :/